src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -772,6 +771,13 @@ public String getFileType(final Map<String, String> metaData, final String forms
         return fileType;
     }
 
+    /**
+     * Parameters that are used to determine filetype in {@link #getFileType(Map, String)}
+     */
+    public enum FileTypeCheckParameter {
+        POPPED_FORMS, COMPLETE_FILETYPE, FILETYPE, FontEncoding, FINAL_ID,
```

Comment:
Use uppercase and create constructor to set the actual values.

Reply:
Fixed